views/metrics/info-card/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box } from '@interest-protocol/ui-kit';
+import { getMetric } from 'api/metrics';
+import { FC, useEffect, useState } from 'react';
+import { v4 } from 'uuid';
+
+import { formatDollars, formatMoney } from '@/utils';
+
+import { TOP_INFO_CARDS_DATA } from '../metrics.data';
+import TopInfoCards from './info-card';
+
+const InfoCardsList: FC = () => {
+  const [data, setData] = useState<any[] | null>(null);
+
+  useEffect(() => {
+    getMetric('get-overview').then(setData);
+  }, []);
+
+  return (
+    <Box
+      gridColumn="1/-1"
```

Comment:
I don't know why we have this line, I think that we don't have a grid as parent, if we have it's ok